website/content/docs/connect/ca/index.mdx

```diff
@@ -211,3 +211,22 @@ servers.
 Currently both Consul and Vault CA providers _do_ support cross signing. As more
 providers are added this documentation will list any that this section applies
 to.
+
+### Recovering From Expired Certificates
+If the built-in CA provider is misconfigured or unavailable, Consul service mesh requests eventually
+stop functioning due to expiration of intermediate and root certificates. To recover manually, use the
+[CLI helper](/commands/tls/ca#consul-tls-ca-create) to generate CA certificates.
+
+
+#### Example - Regenerating the built in CA
+```shell-session
+$ consul tls ca create -cluster-id test -common-name "Consul Agent CA" -days=365 -domain consul
+  ==> Saved consul-agent-ca.pem
+  ==> Saved consul-agent-ca-key.pem
+```
+The example above generates a new CA with a validity of 365 days. The cluster-id argument is specific
+to each cluster and can be looked up by examining the existing Consul Connect CA.
```

Comment:
You may want to point out that the cluster id is in the `TrustDomain` field in the `v1/connect/ca/roots` endpoint

Reply:
This command outputs the trust domain: `curl -s localhost:8500/v1/connect/ca/roots | jq -r '.TrustDomain'

We should really add a `consul connect ca roots` command for retrieving the roots.